src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -84,13 +84,22 @@ protected function getFontsSectionContent()
      */
     protected function getFillsSectionContent()
     {
-        return <<<EOD
-<fills count="1">
-    <fill>
-        <patternFill patternType="none"/>
-    </fill>
-</fills>
-EOD;
+        $registeredStyles = $this->getRegisteredStyles();
+        $fillsCount = count($registeredStyles);
+        $content = sprintf('<fills count="%d">', $fillsCount);
+        foreach ($registeredStyles as $style) {
+            if ($style->shouldApplyBackgroundColor()) {
+                $content .= sprintf(
+                    '<fill><patternFill patternType="solid"><fgColor rgb="%s"/><bgColor rgb="%s"/></patternFill></fill>',
+                    $style->getBackgroundColor(),
```

Comment:
I got this from LibreOffice - They create the border definition for xlsx in that way. I actually have no Excel License :D so I can not check there.